### DISCUSSION TITLE
Getting 404 errors  in sub components

### DISCUSSION THREAD

**Discussion by tonybyng** [livewire/livewire#4005] [7. Bugs (archived)]
Hi

Ive seen several bugs listed about localisation issues with dynamic path changes and 404 errors but mine isnt quite the same

Ive got a livewire form which is working fine. However within the form I have another livewire component and when that interacts I get

Request URL: http://0.0.0.0/livewire/message/entry.entity-form
Request Method: POST
Status Code: 404 Not Found

and a great big white 404 on screen.

The page itself is just running from http://0.0.0.0/entity/5/edit - no custom url that might change

This used to be fine around 2.3 ish (probably the versions that other people have said they were fine with) . I upgraded to 2.4 and had the issue and just upgraded to 2.6.7 and still have the issue (Ive been working on other areas of the system and didnt spot it until the other day)

Just to confirm, if I change an input box in the main form I successfully send a POST to http://0.0.0.0/livewire/message/entry.entity-form and everything works, its only when I interact with the component on the form that its causing an issue.

Can anyone help ?

Thanks

Tony

**Comment by joshhanley** [comment]
@tonybyng that's a hard one to troubleshoot without seeing the setup. Any chance you could create a small example repo which reproduces the issue with as little code as possible?

Thanks!

**Comment by tonybyng** [comment]
Will do. Thanks

**Reply by tonybyng** [reply]
@joshhanley thanks for the reply. The problem isnt where i thought it was so I guess the title of this is now incorrect but I still have the issue. 

I have a component which searches for users so that as you type, it limits the possible matches and as soon as you type something, the 404 is firing, referring to the parent component not the child.  The child emits an event to tell the parent to update.  Ive put a dd() within the parent event listener but its triggering the 404 before it gets there, so its after the  click handler is fired on the child and before the event is handled on the parent.  

Because its part of a big system I aimed to cut the component functionality down to bare bones for the example repo and then couldnt recreate it but I hacked out a whole pile of stuff  so unless you've got any suggestions I think by best solution is to rewrite it from scratch and see if I can reproduce the error in the new version. (I should say this did used to work in an earlier version of livewire so whatever the issue is seems to have crept in about the time when others started reporting 404 issues but I just don't know yet whats causing it)

**Reply by joshhanley** [reply]
@tonybyng no worrries 🙂

Yeah, if an update did start causing the issue, we'd be happy to look at it and try and fix it. But as you said, if you can't replicate in a test, that makes it hard to troubleshoot.

If you do end up rebuilding and find you run into the same issue again, we can dig further and see if we can work out exactly what's causing it.

Thanks!
